docs/customization/controller.rst

```diff
@@ -0,0 +1,142 @@
+Customizing Controllers
+=======================
+
+All **Sylius** resources are using the `` Sylius\Bundle\ResourceBundle\Controller\ResourceController`` as default. but some of them have been already extended in Bundles.
+If you want to override some controller action check which controller you should be extending.
+
+.. note::
+    There are two types of controllers we can define in Sylius.
+    **Resource Controllers** - are basing only on one Entity, so they return only the resources they have in their name. For instance ProductController should return only Products.
+    **Frontend Controllers** - these may use many entities at once, they are useful on more general pages. The best example here is the HomepageController that may return many different objects, such as products, banners, blog posts etc.
```

Comment:
Standard? I'd like to go away from frontend/backend naming and keep shop/admin, but here it is generally about non-resource controllers. WDYT?